RocketChat/Rocket.Chat.ReactNative
# Chore: Hooks app/views/DefaultBrowserView

## Proposed changes

Migrate DefaultBrowserView to hooks

## Issue(s)	


## How to test or reproduce

- After logged
- Join a room, click on any link and see how the web page is opened
- Then, go to Settings and click on `Default browser`
- Change to another type of browser, back to that room in step 2 and open again the link

## Screenshots

## Types of changes




## Checklist



## Further comments
